flybywiresim/aircraft
# fix(apu): Added xfeed APU capabilities

Fixes #[issue_no]

## Summary of Changes

This PR fixes/ adds cross feed APU fuel capabilities.

With xfeed OFF, APU will always burn fuel from the left tank.
With xfeed ON, APU burns fuel from both tanks as per the table below


## Screenshots (if necessary)


![image](https://github.com/flybywiresim/aircraft/assets/70781153/caad17bd-6e61-4652-8aa3-8a7aa3ac44ba)


## References





## Additional context



Discord username (if different from GitHub): TazX [Z+1]#0405

## Testing instructions

Starting with the aircraft C&D:

1. Proceed to start the APU
2. Press Xfeed PB to ON
3. Check fuel is burnt from both Left & Right tanks
4. Turn both Left fuel pumps OFF; check fuel is burnt from Right tank
5. Turn both Left pumps ON
6. Turn both Right pumps OFF; check fuel is burnt from Left tank
7. Turn all fuel pumps OFF; check fuel is burnt from both Left & Right tanks


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
